### PR TITLE
DOC: Fix documentation pull requests.

### PR DESCRIPTION
**Pull request by charris**
This fixes the following documentation PRs, now closed:

- #7942
- #8001 
- #8023

### CODE REVIEW

File: doc/source/user/c-info.python-as-glue.rst [hunk 1]
```diff
@@ -902,27 +902,32 @@ The ``code.c`` file also contains the function ``dfilter2d``:
 
 .. code-block:: c
 
-    /* Assumes b is contiguous and
-       a has strides that are multiples of sizeof(double)
-    */
+    /*
+     * Assumes b is contiguous and * a has strides that are multiples of
```
Comment (rgommers) [inline doc/source/user/c-info.python-as-glue.rst]: typo ``* a``

File: numpy/fft/fftpack.py [hunk 2]
```diff
@@ -467,34 +467,36 @@ def irfft(a, n=None, axis=-1, norm=None):
...
+
     Returns
     -------
     out : ndarray
         The truncated or zero-padded input, transformed along the axis
         indicated by `axis`, or the last one if `axis` is not specified.
         The length of the transformed axis is `n`, or, if `n` is not given,
-        ``2*(m-1)`` where ``m`` is the length of the transformed axis of the
-        input. To get an odd number of output points, `n` must be specified.
+        ``2*m - 2)`` where ``m`` is the length of the transformed axis of
```
Comment (rgommers) [inline numpy/fft/fftpack.py]: either lost a bracket on the way or left one in by accident

**Comment by rgommers** [comment]
LGTM except for the two typos

**Comment by charris** [comment]
Fixed. Thanks for the quick review Ralf.

**Pull request merged**